examples/step-86/doc/intro.dox

```diff
@@ -0,0 +1,440 @@
+    +
+    AU(t)
+    -
+    F(t)
+  }_{=:R(t,U,\dot U)}
+  =
+  0.
+@f}
+This matches the form $R(t,U,\dot U) = 0$ you can find in the
+documentation of PETScWrappers::TimeStepper if you identify the time
+dependent function $y=y(t)$ used there with our solution vector
+$U(t)$, and our notation $R(t,U,\dot U)$ instead of the $F(t,y,\dot y)$
+used there and which we rename because we want to use $F$ as the right
+hand side vector of the ODE indicating forcing terms.
+
+This program uses the PETScWrappers::TimeStepper class, and so we will
+take the latter viewpoint. (It is worth noting that SUNDIALS also has
+a package that can solve ODEs in implicit form, wrapped by the
+SUNDIALS::IDA class.) In what follows, we will continue to use $U(t)$
+as the function we seek, even though the documentation of the class
+uses $y(t)$.
+
+
+<h3> %Mapping the differential equation formulation to the time stepper</h3>
+
+Having identified how we want to see the problem (namely, as an "implicit"
+ODE), the question is how we describe the problem to the time stepper.
+Conceptually, all of the wrappers for time stepping packages we support
+in deal.II only requires us to provide them with a very limited set of
+operations. Specifically, for the implicit formulation used by
+PETScWrappers::TimeStepper, all we need to implement are functions
+that provide the following:
+- A way, for a given $t,U,\dot U$, to evaluate the residual vector
+  $R(t,U,\dot U)$.
+- A way, for a given $t,U,\dot U, \alpha$, to set up a matrix
+  $J := \dfrac{\partial R}{\partial y} +
+  \alpha \dfrac{\partial R}{\partial \dot y}$. This is often
+  called the "Jacobian" of the implicit function $R$, perhaps with
+  a small abuse of terminology. In the current case, this matrix
+  is $J=A + \alpha M$. If you have read through step-26, it is probably
+  not lost on you that this matrix appears prominently there as well --
+  with $\alpha$ being a multiple of the inverse of the time step $k_n$.
+  Importantly, for the linear problem we consider here, $J$ is a linear
+  combination of matrices that do not depend on $U$.
+- A way to solve a linear system with this matrix $J$.
+
+That's really it. If we can provide these three functions, PETSc will do
+the rest (as would, for example, SUNDIALS::ARKode or, if you prefer
+the implicit form, SUNDIALS::IDA). It will not be
+very difficult to set these things up. In practice, the way this will
+work is that inside the `run()` function, we will set up lambda functions
+that can access the information of the surrounding scopes and that
+return the requested information.
+
+In practice, we often want to provide a fourth function:
+- A callback that is called at the end of each time step (or perhaps
+  at other intervals) and that is provided with the current solution
+  and other information and that can be used to "monitor" the progress
+  of computations. One of the ways in which this can be used is to
+  output visualization data every few time steps.
```

Comment:
```suggestion
- A callback that is called at the end of each time step
  and that is provided with the current solution
  and other information that can be used to "monitor" the progress
  of computations. One of the ways in which this can be used is to
  output visualization data every few time steps.
```